### PR TITLE
update release note for Restrictions

### PR DESCRIPTION
**Pull request by YuzuruUeda**

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -19,6 +19,11 @@ Enhancements
 * Allow users to delete multiple migrations
 * 9057 Remove End-of-life Microsoft Azure Classic
 
+Restrictions
+~~~~~~~~~~~~
+* Because of the following issue, the image generation of Linux system to migrate to K5 is restricted.
```
Comment (emuus) [inline changelog.rst]: I would rewrite: Currently the image generation of Linux system for K5 fails. This is due to a known issue during dependency checking.

Comment (YuzuruUeda) [reply changelog.rst]: It seems that generation from a pure appliance template is OK.

**Comment by emuus** [comment]
Yes, this is fine for me

File: changelog.rst (hunk 1 above)
Comment (emuus) [reply changelog.rst]: Yes, this is fine for me

Comment (YuzuruUeda) [reply changelog.rst]: Thank you, Elaine-san.
I will update.

File: changelog.rst [hunk 2]
```diff
@@ -19,6 +19,11 @@ Enhancements
 * Allow users to delete multiple migrations
 * 9057 Remove End-of-life Microsoft Azure Classic
 
+Restrictions
+~~~~~~~~~~~~
+* Currently the image generation of Linux system for K5 migration fails. This is due to a known issue during dependency checking.
+  9941 Image generatin for K5 failed while checking the dependencies
```
Comment (thachmai) [inline changelog.rst]: "Image generati**o**n", I fixed the ticket in jira

Comment (YuzuruUeda) [reply changelog.rst]: Thank you, Thach-san!
I will fix the sentence.

**Pull request merged**